.github/workflows/ci.yml

```diff
@@ -92,6 +92,14 @@ jobs:
   #       token: ${{ secrets.GITHUB_TOKEN }}
   #       args: --all-features -- -D warnings
 
+  docker:
+    name: Build / Docker
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: sudo docker build --target builder .
```

Comment:
We don't use the `sudo` command in the [release-docker](https://github.com/dotenv-linter/dotenv-linter/blob/master/.github/workflows/release.yml#L59) job. Is it necessary here?

Reply:
good point, It is not needed. `sudo` has been removed from the command.